capture/d2l-capture-producer/d2l-capture-producer.js

```diff
@@ -1196,6 +1199,38 @@ class CaptureProducer extends RtlMixin(InternalLocalizeMixin(LitElement)) {
 		this._updateVideoTime();
 	}
 
+	_renderRevisionsDropdownItems() {
+		if (this._revisionsLatestToOldest) {
+			return this._revisionsLatestToOldest.map((revision, index) => {
+				let label;
+				if (index === 0) {
+					label = this.localize('currentDraft');
+				} else {
+					label = `${this.localize('revisionNumber', { number: this._revisionsLatestToOldest.length - index })}`;
+				}
+				return html`<d2l-menu-item data-revision-index="${index}" text="${label}"></d2l-menu-item>`;
+			});
+		} else {
+			return '';
+		}
+	}
+
+	_renderSavedUnsavedIndicator() {
+		let contents;
+		if (this._unsavedChanges) {
+			contents = html`
+				<p class="d2l-label-text" style="color: var(--d2l-color-feedback-warning);">${this.localize('unsaved')}</p>
+				<d2l-icon class="d2l-video-producer-saved-unsaved-indicator-icon" icon="tier1:alert" style="color: var(--d2l-color-feedback-warning);"></d2l-icon>
+			`;
+		} else {
+			contents = html`
+				<p class="d2l-label-text" style="color: var(--d2l-color-feedback-success);">${this.localize('saved')}</p>
+				<d2l-icon class="d2l-video-producer-saved-unsaved-indicator-icon" icon="tier1:check-circle" style="color: var(--d2l-color-feedback-success);"></d2l-icon>
+			`;
+		}
+		return html`<div class="d2l-video-producer-saved-unsaved-indicator-container">${contents}</div>`;
```

Comment:
Minor: Alternatively, we could move some of the differences to variables I think? - e.g., 

```suggestion
  let icon = 'tier1:check-circle';
  let feedbackColor = 'var(--d2l-color-feedback-success)';
  let langterm = 'saved';
    
  if (this._unsavedChanges) {
    feedbackColor = 'var(--d2l-color-feedback-warning)';
    langterm = 'unsaved';
    icon = 'tier1:alert';
  }
  return html`
    <div class="d2l-video-producer-saved-unsaved-indicator-container">
      <p class="d2l-label-text" style="color: ${feedbackColor};">
        ${this.localize(langterm)}
      </p>
      <d2l-icon class="d2l-video-producer-saved-unsaved-indicator-icon" icon="${icon}" style="color: ${feedbackColor};"></d2l-icon>
    </div>`;
```

I think this reduces the duplication a bit but might be negligible - happy with what you have as well